frontend/src/InteractiveImport/Interactive/InteractiveImportRow.js

```diff
@@ -366,7 +368,34 @@ class InteractiveImportRow extends Component {
 
         <TableRowCell>
           {
-            rejections && rejections.length ?
+            customFormats.length ?
+              <Popover
+                anchor={
+                  <Icon name={icons.INTERACTIVE} />
+                }
+                title="Formats"
+                body={
+                  <div className={styles.customFormatTooltip}>
+                    {
+                      customFormats.map((format) => {
```

Comment:
Use `EpisodeFormats` component here to avoid repeating this code. It also handles null formats properly.